src/msgs/MachineInstructions.proto

```diff
@@ -62,6 +62,7 @@ message PrepareInstructionBS {
 
 message PrepareInstructionDS {
   required uint32 gate = 1;
+  required uint32 order_id = 2;
```

Comment:
True, but the rules currently still say that the gate also has to be sent as part of the message. It's not necessary, but it would require a rule change first.

Reply:
Something we could discuss in our next meeting then.

Reply:
That has been settled in the TC meeting: https://github.com/robocup-logistics/rcll-rulebook/pull/21